dev-util/trace-cmd/trace-cmd-2.7.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=(python2_7)
+
+inherit eutils linux-info python-single-r1 toolchain-funcs
```

Comment:
As far as I can see ```eutils``` isn't used anyway and could be removed. Same applies for the ```9999``` ebuild

Reply:
Thanks for noticing, I've updated this PR accordingly.